### DISCUSSION TITLE
Welcome to pywebcopy Discussions!

### DISCUSSION THREAD

**Discussion by rajatomar788** [rajatomar788/pywebcopy#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BradKML** [comment]
Nice to see a competitor of both wget, [HTTrack](https://github.com/xroche/httrack), [ArchiveBox](https://github.com/ArchiveBox/ArchiveBox) and these Golang-based tools:
- https://github.com/antsanchez/go-download-web
- https://github.com/imthaghost/goclone
- https://github.com/cornelk/goscrape

since those require some kind of importing into Python before data science can be done.

**Reply by rajatomar788** [reply]
Well, thanks.
